src/sentry/runner/commands/devserver.py

```diff
@@ -183,7 +183,8 @@ def devserver(
     needs_https = parsed_url.scheme == "https" and (parsed_url.port or 443) > 1024
     has_https = shutil.which("https") is not None
 
-    control_silo_port = port + 10
+    control_silo_port = port
+    region_silo_port = port + 10
```

Comment:
I think you'll also need to change webpack configuration (webpack.config.ts) as it also uses port numbers when proxying frontend requests. Currently webpack is reading SENTRY_BACKEND_PORT and expecting that to be the region server.